docs/src/ci.md

```diff
@@ -59,14 +59,397 @@ export default defineConfig({
 });
 ```
 
-
 ## CI configurations
 
+### GitHub Actions
+
 The [Command line tools](./browsers#install-system-dependencies) can be used to install all operating system dependencies on GitHub Actions.
```

Comment:
I don't understand who this sentence is addressed to under `GitHub Actions` title. Previously the user was redirected to our comprehensive getting started guide which is where they should start. If they know that stuff, they likely now how to install browsers and deps already. I'd revert `Check out our [GitHub Actions](ci-intro.md) guide` here or maybe just drop this paragraph.

Reply:
This PR moves the comprehensive github actions guide into here, so `ci-intro.md` is just an intro. I agree that the paragraph reads weird though, max was saying the same in https://github.com/microsoft/playwright/pull/31988#discussion_r1701947754. I'll move the paragraph above the heading and reword.